src/System.Runtime.Loader/tests/ContextualReflectionDependency/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<Project DefaultTargets="Build">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp-Unix;
+      netcoreapp-Windows_NT;
```

Comment:
This does not have any OS-specific code. It does not need to be build OS specific. (Also, we prefer to not have OS-specific builds for tests.)

Reply:
I was copying from 
https://github.com/dotnet/corefx/blob/8b515adcd4ea0b8233f018f9e87ef946ce0e7c42/src/System.Runtime.Loader/tests/Configurations.props#L4-L5

I'll fix this library.